OneBlink.SDK/models/FormElement.cs

```diff
@@ -718,5 +723,50 @@ private static Guid initialiseId(Guid? id)
             }
             return civicaStreetNameElement;
         }
+        public static FormElement CreateFilesElementEvent(
+        string name,
+        string label,
+        Guid? id = null,
+        bool readOnly = false,
+        string hint = null,
+        string storageType = null,
+        bool restrictFileTypes = false,
+        List<string> restrictedFileTypes = null,
+        dynamic defaultValue = null,

```

Comment:
I think we leave this property out for now. If we did want to add it we would probably want a class specifically for this type.